policykit/policyengine/safe_exec_code.py

```diff
@@ -36,6 +37,17 @@ def visit_Import(self, node):
     visit_ImportFrom = visit_Import
 
 
+def _hook_writable(obj):
+    """Only allow writing to lists and dicts."""
+    # if obj.__class__.__name__ == "MY_CLASS_NAME":
+    #     return obj
```

Comment:
If we ever want to support assigning attributes to certain objects (like, `proposal.status = FAILED` we could permit those objects here

Reply:
I think definitely proposal.status should be allowed to be assigned. Any others that come to mind?

Reply:
No, right now I can't think of any reason to allow assigning attributes on objects. So it is completely restricted for now.